app/code/Magento/Catalog/Block/Adminhtml/Product/Helper/Form/Gallery/Content.php

```diff
@@ -50,22 +51,30 @@ class Content extends \Magento\Backend\Block\Widget
      */
     private $imageUploadConfigDataProvider;
 
+    /**
+     * @var Database
+     */
+    private $fileStorageDatabase;
+
     /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Framework\Json\EncoderInterface $jsonEncoder
      * @param \Magento\Catalog\Model\Product\Media\Config $mediaConfig
+     * @param Database $fileStorageDatabase
      * @param array $data
      * @param ImageUploadConfigDataProvider $imageUploadConfigDataProvider
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Framework\Json\EncoderInterface $jsonEncoder,
         \Magento\Catalog\Model\Product\Media\Config $mediaConfig,
+        Database $fileStorageDatabase,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
As requested (and mental note made for future PR's)